src/html_builder.ts

```diff
@@ -15,486 +15,499 @@ import {
     get_init_style
 } from '../src/blobs';
 
-export class HTMLBuilder {
 
-    static add_style_to_document(ulabel: ULabel) {
-        let head: HTMLHeadElement = document.head || document.getElementsByTagName('head')[0];
-        let style: HTMLStyleElement = document.createElement('style');
-        head.appendChild(style);
-        
-        style.appendChild(document.createTextNode(get_init_style(ulabel.config["container_id"])));
-        
-    }
 
-    private static get_md_button(md_key, md_name, svg_blob, cur_md, subtasks) {
-        let sel: string = "";
-        let href: string = ` href="#"`;
-        if (cur_md == md_key) {
-            sel = " sel";
-            href = "";
-        }
-        let subtask_classes: string = "";
-        for (const st_key in subtasks) {
-            if (subtasks[st_key]["allowed_modes"].includes(md_key)) {
-                subtask_classes += " md-en4--" + st_key;
-            }
-        }
+export function add_style_to_document(ulabel: ULabel) {
+    let head: HTMLHeadElement = document.head || document.getElementsByTagName('head')[0];
+    let style: HTMLStyleElement = document.createElement('style');
+    head.appendChild(style);
+    
+    style.appendChild(document.createTextNode(get_init_style(ulabel.config["container_id"])));
+}
 
-        return `<div class="mode-opt">
-            <a${href} id="md-btn--${md_key}" class="md-btn${sel}${subtask_classes} invert-this-svg" amdname="${md_name}">
-                ${svg_blob}
-            </a>
-        </div>`;
+function get_md_button(md_key, md_name, svg_blob, cur_md, subtasks) {
```

Comment:
Im dumb, what is `md`? mode? Maybe add some brief function descriptions

Reply:
Yeah, `md` stands for mode. Definitely not a bad idea to add more function descriptions.

Reply:
Added function descriptions [1b40b63](https://github.com/SenteraLLC/ulabel/pull/127/commits/1b40b6346fb434a25ac0da280be20c559525b821)